docs/quickstarts/nextjs.mdx

```diff
@@ -40,62 +68,66 @@ CLERK_SECRET_KEY={{secret}}
 
 </InjectKeys>
 
-### Mount `<ClerkProvider />`
-
-Add the [`<ClerkProvider />`][clerk-provider] wrapper to your application. The [`<ClerkProvider />`][clerk-provider] component wraps your Next.js application to provide active session and user context to Clerk's hooks and other components. A reasonable default approach is to have [`<ClerkProvider />`][clerk-provider] wrap the `<body/>` to enable the context to be accessible anywhere within the app.
+### Wrap your app in `<ClerkProvider />`
 
+The [`<ClerkProvider />`](/docs/components/clerk-provider) component provides active session and user context to Clerk's hooks and other components. For this project, it makes sense to wrap the `<html/>` element with `<ClerkProvider />`. This makes the active session and user context accessible anywhere within the app.
 
-<Callout type="info">
-  The [`<ClerkProvider />`][clerk-provider] component needs to access [headers](https://nextjs.org/docs/app/api-reference/functions/headers) to authenticate correctly. This means anything wrapped by the provider will be opted into [dynamic rendering](https://nextjs.org/docs/app/building-your-application/rendering/server-components#dynamic-rendering) at request time. If you have static-optimized or ISR pages that you would prefer not to be opted into dynamic rendering, make sure they are <em>not</em> wrapped by [`<ClerkProvider />`][clerk-provider].
-  
-  This is easiest to accomplish by ensuring that [`<ClerkProvider />`][clerk-provider] is added further down the tree to wrap route groups that explicitly need authentication instead of having the provider wrap your application root as recommended above. For example, if your project includes a set of landing/marketing pages as well as a dashboard that requires login, you would create separate `(marketing)` and `(dashboard)` route groups. Adding [`<ClerkProvider />`][clerk-provider] to the `(dashboard)/layout.jsx` layout file will ensure that only those routes are opted into dynamic rendering, allowing the marketing routes to be statically optimized.
+<Tabs items={["App Router version", "Page Router version"]}>
+  <Tab>
+    
+    <Callout>
+      The `app/layout.tsx` file might be in your `src` folder if it isn't in your root folder.
+    </Callout>
 
-</Callout>
+    ```tsx filename="/src/app/layout.tsx"
+    import { ClerkProvider } from '@clerk/nextjs'

```

Comment:
```suggestion
import { ClerkProvider } from '@clerk/nextjs'
import type { Metadata } from 'next'
import { Inter } from 'next/font/google'
import './globals.css'

const inter = Inter({ subsets: ['latin'] })
```

Reply:
I feel pretty strongly we should keep at least the import for globals.css. If people using the quickstart use the 'copy' button then that disables their Tailwind/CSS and they may not realize it. 

I think there is value in importing Inter, but also fine with not having it. The new Quickstart repo should be adjusted to match the final version of this.